core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1939,7 +1941,54 @@ public static boolean isValidTablePropertyKey(String key) {
 
       // others
       TSERV_NATIVEMAP_ENABLED, TSERV_MAXMEM, TSERV_SCAN_MAX_OPENFILES,
-      MANAGER_RECOVERY_WAL_EXISTENCE_CACHE_TIME, COMPACTION_COORDINATOR_FINALIZER_QUEUE_SIZE);
+      MANAGER_RECOVERY_WAL_EXISTENCE_CACHE_TIME, COMPACTION_COORDINATOR_FINALIZER_QUEUE_SIZE,
```

Comment:
Adding all the new Properties added to FIXED_PROPERTIES at the end of the list, for easier review. Will organize these in a later commit.

Reply:
Organized in bf12f6c6731c0a271ac0a5ba2301ba29553c0e54